### PR TITLE
Make template outerHTML work in IE

### PR DESCRIPTION
**Pull request by trevordixon**
#474

### CODE REVIEW

File: src/Template/Template.js [hunk 1]
```diff
@@ -37,29 +37,51 @@ if (typeof HTMLTemplateElement === 'undefined') {
...
-              for (var e = this.content.firstChild; e; e = e.nextSibling) {
-                o += e.outerHTML || escapeData(e.data);
-              }
-              return o;
+          Object.defineProperties(template, {
+            innerHTML: {
+              get: function() {
+                var o = '';
+                for (var e = this.content.firstChild; e; e = e.nextSibling) {
+                  if (e.tagName == 'TEMPLATE') { console.log(e.outerHTML); }
```
Comment (azakus) [inline src/Template/Template.js]: This looks like a leftover debug line.

File: src/Template/Template.js [hunk 2]
```diff
@@ -37,29 +37,51 @@ if (typeof HTMLTemplateElement === 'undefined') {
...
+                var attrs = this.attributes;
+                for (var i = 0, attr; attr = attrs[i]; i++) {
+                  s += ' ' + attr.name + '="' + escapeAttr(attr.value) + '"';
+                }
+                return s + '>' + this.innerHTML + '</template>';
+              },
+              set: function(value) {
+                var p = this.parentNode;
+                if (p) {
+                  p.invalidateShadowRenderer();
```
Comment (azakus) [inline src/Template/Template.js]: This should check that `invalidateShaddowRenderer` is a function first.

**Comment by azakus** [comment]
Please add a test of `outerHTML`, fix the issues in the comments, then I'll merge. Thanks!

**Comment by addyosmani** [comment]
Friendly ping @trevordixon

**Comment by trevordixon** [comment]
Thank @addyosmani. How does that look?

**Comment by TimvdLippe** [comment]
This was implemented in https://github.com/webcomponents/template/pull/22

**Pull request closed without merging**